openbb_terminal/portfolio/portfolio_controller.py

```diff
@@ -356,9 +356,9 @@ def call_load(self, other_args: List[str]):
                 f"\n[bold][param]Portfolio:[/param][/bold] {self.portfolio_name}"
             )
 
-            self.risk_free_rate = ns_parser.risk_free_rate / 100
+            self.risk_free_rate = ns_parser.risk_free_rate
             console.print(
-                f"[bold][param]Risk Free Rate:[/param][/bold] {self.risk_free_rate:.2%}"
+                f"[bold][param]Risk Free Rate:[/param][/bold] {self.risk_free_rate}%"
```

Comment:
What is the desired behaviour?  If I enter 3.  Is that 3% -> 0.03?

If so then we want to keep the /100 and print 100*rate% ?

Reply:
The original intention was to put it in %. So if you put in 5 it would convert it to 5%.  The proxy for the risk-free rate tends to be pretty low (0% - 5%) meaning that you'd need to enter a bunch of zeros anyways. In 2020 at some point it was 0.65% which would be 0.0065 in the Terminal.

If we do not have this behaviour anywhere else in the Terminal perhaps 0.0065 would still be better though.

Reply:
> If so then we want to keep the /100 and print 100*rate% ?

The problem is that every other command also divides the risk free rate input by hundred, thus it would divide the default risk free rate twice:
Input: 5  ---(Saves to)---> 0.05 ---(Gets executed as)---> 0.0005; thus resulting in the incorrect result